src/backend/src/utils/errors.utils.ts

```diff
@@ -0,0 +1,53 @@
+import { ErrorRequestHandler, Request, Response, NextFunction } from 'express';
+
+export class Exception extends Error {
+  public status: number;
+
+  constructor(status: number, message: string) {
+    super(message);
+    this.status = status;
+  }
+}
+
+export const errorHandler: ErrorRequestHandler = (error: unknown, req: Request, res: Response, next: NextFunction) => {
+  if (res.headersSent) {
+    return next(error);
+  }
+
+  if (error instanceof Exception) {
+    res.status(error.status).json({ message: error.message });
+  } else {
+    res.status(400).json({ message: 'Something went very wrong...' });
```

Comment:
Why would we make this a 400 if something went very wrong? That sounds like a 500 - server side error to me?